### PR TITLE
Add project denmark

### PR DESCRIPTION
**Pull request by welrachid**
Hi
I've now added denmark

I've used norway as a starting point.

I've had issues with making it work without whosonfirst (which is a project i didnt know before). This kept giving me an elastic search size ofunder 10MB, but after i enabled it, i've been able to retrieve data as expected (and elastic search is now >1GB

i'm used to work with gitlab where i can squash.. not sure how this works, but ofcourse you will see my previous "master" changes in this. 

Hopefully someone finds it useful

### CODE REVIEW

File: projects/denmark/pelias.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    },
+    "polyline": {
+      "datapath": "/data/polylines",
+      "files": [ "extract.0sv" ]
+    },
+    "whosonfirst": {
+      "datapath": "/data/whosonfirst",
+      "countryCode": "DK",
+      "importPostalcodes": true,
+      "importPlace": [ "101749163" ]
```
Comment (missinglink) [inline projects/denmark/pelias.json:59]: Can you please change the ID from [101749163](https://spelunker.whosonfirst.org/id/101749163/) to [85633121](https://spelunker.whosonfirst.org/id/85633121/), then I'm happy to merge this.

**Review by missinglink — CHANGES_REQUESTED** [review]

File: projects/denmark/pelias.json (hunk 1 above)
Comment (welrachid) [reply projects/denmark/pelias.json]: I've added the correct id. Thx

**Comment by welrachid** [comment]
Should be updated now
let me know if you need me to do anything else

**Comment by missinglink** [comment]
Thanks!

**Pull request merged**